Anytype/Sources/ServiceLayer/SyncStatus/SyncStatusStorage.swift

```diff
@@ -1,31 +1,34 @@
 import Combine
 import Services
 import AnytypeCore
+import ProtobufMessages
 
 
 protocol SyncStatusStorageProtocol {
-    var statusPublisher: AnyPublisher<SyncStatus, Never> { get }
-    var currentStatus: SyncStatus { get }
+    func statusPublisher(spaceId: String) -> AnyPublisher<SyncStatus, Never>
     
-    func startSubscription(spaceId: String)
+    func startSubscription()
     func stopSubscriptionAndClean()
 }
 
 final class SyncStatusStorage: SyncStatusStorageProtocol {
-    var statusPublisher: AnyPublisher<SyncStatus, Never> { $_status.eraseToAnyPublisher() }
-    var currentStatus: SyncStatus { _status }
-    
-    private var spaceId: String?
-    
-    @Published private var _status: SyncStatus = .offline // TODO: Remove
-    
+    @Published private var _update: Anytype_Event.Space.SyncStatus.Update?
+    private var updatePublisher: AnyPublisher<Anytype_Event.Space.SyncStatus.Update?, Never> { $_update.eraseToAnyPublisher() }
     private var subscription: AnyCancellable?
     
+    private var defaultValues = [String: SyncStatus]()
+    
     nonisolated init() { }
     
-    func startSubscription(spaceId: String) {
-        self.spaceId = spaceId
-        
+    func statusPublisher(spaceId: String) -> AnyPublisher<SyncStatus, Never> {
+        updatePublisher
```

Comment:
Is not difficult code? You can mark `defaultValues` as publisher and some screens can be subscribe for all states. And `statusPublisher` impl it will be easy